module/enrichers.mjs

```diff
@@ -823,6 +903,36 @@ async function rollAction(event) {
         case "tool":
           options.ability = ability;
           return await actor.rollToolCheck(tool, options);
+        case "item":
+          if ( target.dataset.rollItemUuid ) {
+            const gameActor = await fromUuid(target.dataset.rollItemActor);
+            if ( gameActor.testUserPermission(game.user, "OWNER") ) {
+              return (await fromUuid(target.dataset.rollItemUuid)).use();
+            } else {
+              return ui.notifications.warn(`You do not have ownership of ${gameActor?.name}, and cannot use this item.`);
+            }
```

Comment:
This permissions check should probably be handled by `Item5e#use` method rather than here.

```suggestion
            return (await fromUuid(target.dataset.rollItemUuid)).use();
```

(in `item.mjs`)
```javascript
  async use(config={}, options={}) {
    if ( !this.isOwner ) {
      ui.notifications.error("DND5E.DocumentUseWarn", { localize: true });
      return null;
    }
```

(in `en.json`)
```json
"DND5E.DocumentUseWarn": "You lack permission to use an item on this document.",
```

Reply:
It looks like with this change non-owners can use an item enricher built with the UUID or Relative ID
It is used from the proper owning actor, but with the player as the Speaker. They can't use the Attack or Damage buttons, but yea it's not preventing them from using it
![image](https://github.com/foundryvtt/dnd5e/assets/86370342/9d0e528a-3e5c-44c2-a675-967c75316002)